README.md

```
@@ -1,337 +1,203 @@
+- Tratamento de erros em operações assíncronas
+
+## Como Executar
+
+### **Pré-requisitos**
+- Flutter SDK 3.8.1+
+- Dart 3.8.1+
+- Dependências instaladas
+
+### **Instalação**
+bash
+# Clonar o projeto
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)